### DISCUSSION TITLE
Axis colors definition

### DISCUSSION THREAD

**Discussion by igennart** [PMEAL/pytrax#21] [Q&A]
Dear, is it possible to know to what corresponds the axis colors of the tortuosity graph ?

Is it this definition ?

X-axis = red, Y-axis = green, Z-axis = blue

Thanks, 
Best regards,

**Comment by TomTranter** [comment]
It relates to the 0, 1, 2 axis of the image as if you were slicing it with numpy. I think this is row, column, depth which might be Y, X, Z if you compare to how it looks on a screen with a standard graphical axis
